src/pages/teams/[id].tsx

```diff
@@ -1,3 +1,68 @@
-import Team from '../../views/Teams/Team'
+import React from 'react'
+// eslint-disable-next-line camelcase
+import { SWRConfig, unstable_serialize } from 'swr'
+import { GetStaticPaths, GetStaticProps, InferGetStaticPropsType } from 'next'
+import TeamPageRouter from 'views/Teams/TeamPageRouter'
+import teams from 'config/constants/teams'
+import { getTeam } from 'state/teams/helpers'
+import { teamsById } from 'utils/teamsById'
 
-export default Team
+const TeamPage = ({ fallback }: InferGetStaticPropsType<typeof getStaticProps>) => {
+  return (
+    <SWRConfig
+      value={{
+        fallback,
+      }}
+    >
+      <TeamPageRouter />
+    </SWRConfig>
+  )
+}
+
+export const getStaticPaths: GetStaticPaths = () => {
+  return {
+    fallback: true,
+    paths: [],
+  }
+}
+
+export const getStaticProps: GetStaticProps = async ({ params }) => {
+  const teamIds = (params.id as string[]) || []
```

Comment:
I'm curious how does it work, `params.id` should be string 🤯

Reply:
Done